osmaxx/excerptexport/models/excerpt.py

```diff
@@ -84,6 +85,13 @@ def color(self):
     def extent(self):
         return self.bounding_geometry.extent
 
+    @property
+    def has_running_exports(self):
+        return any([not i.can_be_deleted for es in self.extraction_orders.all() for i in es.exports.all()])
```

Comment:
Please use variable name `i` only for integer-valued counters, integer-valued indices etc.